src/main/java/de/tum/in/www1/artemis/config/Constants.java

```diff
@@ -7,6 +7,14 @@
  */
 public final class Constants {
 
+    public static final int USERNAME_MIN_LENGTH = 4;
+
+    public static final int USERNAME_MAX_LENGTH = 50;
+
+    public static final int PASSWORD_MIN_LENGTH = 4;
```

Comment:
We could go for 8 in a follow up PR, at the moment I would keep it to 4 to avoid breaking some cases (we don't know if there might be an existing user with a password that is 7 characters long).